frontend/src/metabase/querying/notebook/components/Notebook/Notebook.tsx

```diff
@@ -69,7 +69,7 @@ const Notebook = ({
 
   return (
     <Box pos="relative" p={{ base: "1rem", sm: "2rem" }}>
-      <NotebookSteps
+      <NotebookStepList
```

Comment:
The old name was better

Reply:
Previously we had `NotebookStep` and `NotebookSteps`. Arguably components shouldn't differ by one character.